src/Common/src/Interop/Windows/mincore/Interop.GetFullPathNameW.cs

```diff
@@ -22,19 +22,25 @@ partial class mincore
         [DllImport(Libraries.CoreFile_L1, EntryPoint = "GetFullPathNameW", SetLastError = true, CharSet = CharSet.Unicode, BestFitMapping = false)]
         private static extern int GetFullPathNamePrivate(string path, int numBufferChars, [Out]StringBuilder buffer, IntPtr mustBeZero);
 
-        internal static int GetFullPathName(string path, int numBufferChars, [Out]StringBuilder buffer, IntPtr mustBeZero)
+        internal static int GetFullPathName(string path, int numBufferChars, [Out]StringBuilder buffer)
```

Comment:
> I could simply remove these from interop and put them directly in the PathInternal.

No, we should keep the P/Invokes in the ```Interop.*.cs``` files.  The point of these files is to have them be callable versions of the OS functions, and nothing more, as if we were able to #include .h files.  The exposed functions should have the same behavior as the OS functions, so that someone calling them can look at MSDN and see the documented behavior.  Functionality above these files can then layer on additional/different behaviors.  Once we start letting arbitrary changes in to these ```Interop.*.cs``` files, they lose their meaning.